_translator-files/po/sv-SE/Tips-Tricks-More.po

```diff
@@ -213,108 +216,131 @@ msgid ""
 
 #. type: Title ##
 #: ../wiki/en/Tips-Tricks-More.md:101
 #, markdown-text, no-wrap
 msgid "For Server admins"
-msgstr ""
+msgstr "För serveradministratörer"
 
 #. type: Title ###
 #: ../wiki/en/Tips-Tricks-More.md:103
 #, markdown-text, no-wrap
 msgid "Converting a Registered Server to an Unregistered one on the fly"
-msgstr ""
+msgstr "Konvertera en registrerad server till en oregistrerad server i farten"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:106
 #, markdown-text
 msgid "You can run as a Registered Server long enough for people to connect, then go \"private\" (Unregistered) by setting the Directory to \"none\" in the Server GUI. Musicians will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)"
-msgstr ""
+msgstr "Du kan köra som en registrerad server tillräckligt länge för att människor ska kunna ansluta och sedan gå till \"privat\" (oregistrerad) genom att ställa in katalogen på \"ingen\" i serverns GUI. Musiker kommer fortfarande att vara anslutna till servern tills de kopplar från. (Tack till [David Savinkoff](https://github.com/DavidSavinkoff) för detta tips!)"
 
 #. type: Title ###
 #: ../wiki/en/Tips-Tricks-More.md:107
 #, markdown-text, no-wrap
 msgid "Remote management of recordings"
-msgstr ""
+msgstr "Fjärrhantering av inspelningar"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:110
 #, markdown-text
 msgid "Jamulus user [vdellamea](https://github.com/vdellamea) has written a [web-based remote tool](https://github.com/vdellamea/jamulus-server-remote) for starting and stopping recordings on Linux Servers, allowing you to then download them from your browser. See also [Jamulus Jam Exporter](https://github.com/pljones/jamulus-jamexporter) by [pljones](https://github.com/pljones), which also includes a Server recording recovery script."
-msgstr ""
+msgstr "Jamulus-användare [vdellamea](https://github.com/vdellamea) har skrivit ett [webbaserat fjärrverktyg](https://github.com/vdellamea/jamulus-server-remote) för att starta och stoppa inspelningar på Linux Servrar, så att du sedan kan ladda ner dem från din webbläsare. Se även [Jamulus Jam Exporter](https://github.com/pljones/jamulus-jamexporter) av [pljones](https://github.com/pljones), som också inkluderar ett serverinspelningsåterställningsskript."
 
 #. type: Title ###
 #: ../wiki/en/Tips-Tricks-More.md:111
 #, markdown-text, no-wrap
 msgid "Making a Server status page"
-msgstr ""
+msgstr "Skapa en serverstatussida"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:114
 #, markdown-text
 msgid "With the `-m` command line argument, Server statistics can be generated to be put on a web page."
```

Comment:
@pljones I think it could be worth discussing deprecating this and integrating it into JSON RPC (not here).